keanu-project/src/main/java/io/improbable/keanu/util/status/StatusBar.java

```diff
@@ -62,7 +62,7 @@ public StatusBar(PrintStream printStream, ScheduledExecutorService scheduler) {
 
         // Status bar is disabled for testing.
         String disableStatusBar = System.getProperty("io.improbable.keanu.util.status.StatusBar.disableStatusBar");
-        if (disableStatusBar != null && disableStatusBar.equals("true")) {
+        if ("true".equals(disableStatusBar)) {
```

Comment:
When we move the String Literal to the left, we can avoid the null check. 
if `disableStatusBar` happens to be `null`, the comparison will evaluate to `false`